spec/features/idv/doc_auth/hybrid_handoff_spec.rb

```diff
@@ -380,10 +380,12 @@ def verify_no_upload_photos_section_and_link(page)
         context 'when sp ipp is available' do
           before do
             expect(page).to have_current_path(idv_how_to_verify_path)
-            click_on t('forms.buttons.continue_remote')
           end
           context 'when selfie is enabled system wide' do
             describe 'when selfie is required by sp' do
+              before do
+                click_on t('forms.buttons.continue_remote_selfie')
+              end
```

Comment:
When selfie is enabled the button text changed- so this original button `continue_remote` was not found.